index.js

```diff
@@ -49,18 +50,32 @@ class BootSimulator {
         });
       }
     });
+    if(this.deviceInfo.length<1){
+      console.log("No device matches with provided device name");
+      device='';
+    }
+    else if(this.deviceInfo.length>1){
     const answer = await inquirer.prompt({
       type: 'list',
       name: 'Device',
       message: 'Select Simulator 📱 to boot?',
       pageSize: 15,
       choices: this.deviceInfo
     });
-    let device = await _.split(answer.Device, '-');
+    device = await _.split(answer.Device, '-');
+  }
+  else{
+    device= await _.split(this.deviceInfo[0],'-');
+
+  }
+
     let udid = await getAllSimulator.findUDIDOfSim(device[0], device[1]);
+    if(udid){
+    console.log("Booting "+device[0]+" with "+device[1]);
     await exec('xcrun simctl boot ' + udid);
     await exec('open /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app/');
```

Comment:
you need to use simctl to open the simulator. What in this case if the user has Xcode.app in custom path

Reply:
I couldn't find any command in simctl to open the simulator. let me know if you know any.